ccloud/docs/ccloud-stack.rst

```diff
@@ -54,8 +54,8 @@ By default, the |ccloud| ksqlDB app is not created with ``ccloud-stack``, you ha
    ccloud api-key create --service-account $SERVICE_ACCOUNT_ID --resource $RESOURCE -o json    // for schema-registry
 
    # By default, ccloud-stack does not enable Confluent Cloud ksqlDB, but if you explicitly enable it:
-   ccloud ksql app create --cluster $CLUSTER -o json "$KSQLDB_NAME"
-   ccloud api-key create --service-account $KSQLDB_SERVICE_ACCOUNT_ID --resource $RESOURCE -o json    // for ksqlDB
+   ccloud ksql app create --cluster $CLUSTER --api-key "$kafka_api_key" --api-secret "$kafka_api_secret" -o json "$KSQLDB_NAME"
```

Comment:
This should use same parameter convention as the rest of the lines obove (capitalized, etc)

Reply:
With this comment, I'm not certain if you're concerned with only the documentation or the code and the doc?  If you're only thinking about the documentation, and you're not concnered with the docs loosely matching the actual code (which it currently does), I suggest we rework this doc section to use sample values instead of variables, which aren't helpful to a user trying grok what commands are actually run. Or, we could more generalize this section to describe the types of behaviors (maybe with links to the actual ccloud docs) that the script performs, as I don't think this list encompasses everything required as it stands now.

WDYT?

Reply:
Purely doc.  All lines above and below use capitalization.

```suggestion
   ccloud ksql app create --cluster $CLUSTER --api-key "$KAFKA_API_KEY" --api-secret "$KAFKA_API_SECRET" -o json "$KSQLDB_NAME"
```

Reply:
staging docs have this change now